src/main/java/bio/terra/flight/study/ingest/StudyIngestFlight.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.flight.study.ingest;
+
+import bio.terra.dao.StudyDao;
+import bio.terra.pdao.bigquery.BigQueryPdao;
+import bio.terra.stairway.Flight;
+import bio.terra.stairway.FlightMap;
+import org.springframework.context.ApplicationContext;
+
+public class StudyIngestFlight extends Flight {
+
+    public StudyIngestFlight(FlightMap inputParameters, Object applicationContext) {
+        super(inputParameters, applicationContext);
+
+        // get the required daos to pass into the steps
+        ApplicationContext appContext = (ApplicationContext) applicationContext;
+        StudyDao studyDao = (StudyDao)appContext.getBean("studyDao");
+        BigQueryPdao bigQueryPdao = (BigQueryPdao)appContext.getBean("bigQueryPdao");
+
```

Comment:
We can't hold anything from the repository metadata in the objects. That is because the metadata can change asynchronously across the steps and across failure/restart of the DR. The only things we can pass for constructor params are application context things.